### PR TITLE
Dpp 1256 1 transfer learner no funds available

### PR DESCRIPTION
**Pull request by tom-gough**

### CODE REVIEW

File: src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/StepDefinitions/EmployerAccountSteps.cs [hunk 1]
```diff
@@ -85,6 +85,12 @@ public void GivenNamedEmployersTransferBalanceIsMoreThanPrice(int employerId)
             UpdateTransferAllowance(employerId, int.MaxValue);
         }
 
+        [Given("the employer (.*) has a transfer allowance = 0 for all months")]
+        public void GivenNamedEmployersTransferBalanceIsZeroForAllMonths(int employerId)
```
Comment (carlbm) [inline src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/StepDefinitions/EmployerAccountSteps.cs]: Is it worth making the 0 a parameter?

**Review by carlbm — APPROVED** [review]

File: src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/StepDefinitions/EmployerAccountSteps.cs (hunk 1 above)
Comment (scottywakefield) [reply src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/StepDefinitions/EmployerAccountSteps.cs]: seems a fair enough comment, your thoughts @tom-gough ?

Comment (tom-gough) [reply src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/StepDefinitions/EmployerAccountSteps.cs]: we will either need:
0 - no transfer balance
int.MaxValue - transfer balance not an issue in this scenario
a table of values - if we are running out of transfer allowance between months

I can't see a scenario where we would ever need to pass in an explicit value which is applicable to all months, it doesn't really make sense given that the allowance will decrement over months.

One thing I have just noticed myself though is that I have named the method TransferBalance which should be punishable by death. I will fix that.

Comment (scottywakefield) [reply src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/StepDefinitions/EmployerAccountSteps.cs]: 😨 
ok

**Review by scottywakefield — APPROVED** [review]

**Pull request merged**